src/control/server/storage/scm/ipmctl.go

```diff
@@ -7,39 +7,31 @@
 package scm
 
 import (
-	"encoding/json"
 	"fmt"
 	"os/exec"
-	"regexp"
-	"strings"
+	"sort"
+	"strconv"
 	"sync"
 
 	"github.com/dustin/go-humanize"
 	"github.com/pkg/errors"
 
+	"github.com/daos-stack/daos/src/control/fault"
 	"github.com/daos-stack/daos/src/control/lib/ipmctl"
 	"github.com/daos-stack/daos/src/control/logging"
 	"github.com/daos-stack/daos/src/control/server/storage"
 )
 
 const (
 	minNrNssPerSocket = 1
-	maxNrNssPerSocket = 8
+	maxNrNssPerSocket = 4
```

Comment:
Mainly because we don't anticipate an immediate need/usecase but also because there hasn't been any testing of higher values and don't want to give the impression that we have validated using higher values.